gsoc_final_post/inigo_tellaetxe_gsoc_2024_final_report.rst

```diff
@@ -0,0 +1,226 @@
+--------
+
+The objective of the project is to generate synthetic human tractograms with tuneable age and neurodegeneration status properties, using deep learning models. The project is inspired by the `FINTA <https://doi.org/10.1016/j.media.2021.102126>`_ and `GESTA <https://doi.org/10.1016/j.media.2023.102761>`_ works, which propose a generative method using an AutoEncoder architecture for unconditional tractogram generation, and a robust sampling strategy, respectively. In our case, we replicated the results of the FINTA paper AutoEncoder in the FiberCup dataset using TensorFlow2 (TF2) as the deep learning backend, instead of the original PyTorch backend. We also investigated how to condition the AutoEncoder architecture on scalar and categorical variables to support conditional tractogram generation on age and neurodegeneration status. Finally, we also explored the use of Variational AutoEncoders, Regression AutoEncoders, and Adversarial AutoEncoders for tractogram generation under the mentioned conditions.
+
+Proposed Objectives
+-------------------
+
+* Replicate the results of the AutoEncoder architecture in the `FINTA <https://doi.org/10.1016/j.media.2021.102126>`_ paper using TensorFlow2 as the deep learning backend.
+* Obtain publicly available data from the `ADNI <http://adni.loni.usc.edu/>`_, the `FiberCup <https://tractometer.org/fibercup/home/>`_ and the `ISMRM2015 tractography challenge <https://tractometer.org/ismrm2015/home/>`_ to compute tractograms for training the deep learning models.
+* Integrate the replicated unconditional AutoEncoder architecture and sampling algorithm into DIPY to generate synthetic human tractograms, after training it with extensive human tractogram data.
+* Investigate how to condition the AutoEncoder architecture on scalar and categorical variables to support conditional tractogram generation on age and neurodegeneration status.
+* Implement a conditional AutoEncoder architecture that can generate synthetic tractograms conditioned on age and neurodegeneration status.
+
+
+Modified Objectives(Additional)
+-------------------------------
+
+* Investigate the use of Variational AutoEncoders (VAE) for unconditional tractogram generation based on the GESTA architecture.
+* Investigate the use of Conditional Variational AutoEncoders (CVAE) for conditional tractogram generation based on the GESTA architecture.
+* Investigate the possibility to condition the tractogram generation on the fiber bundle for additional control over the process and the data generation, using Adversarial AutoEncoders with attribute-based regularization.
+
+Objectives Completed
+--------------------
+
+* **Literature Review on synthetic tractography generation using AutoEncoders**. Main inspirations: 
+
+  * The `FINTA <https://10.1016/j.media.2021.102126>`_ and `FINTA <https://doi.org/10.1016/j.media.2021.102126>`_ papers because they provide a relatively simple AE architecture with an open source sampling algorithm, easy to reuse.
```

Comment:
One should be GESTA right? And the first link seems to be wrong.

Reply:
There should only be one link to FINTA, we did dive into GESTA.